### PR TITLE
New guided install overview

### PR DESCRIPTION
**Pull request by paperclypse**
### Tell us why

The guided install overview doc.

### Anything else you'd like to share?

I'm not super happy with how the button styles work. I'm thinking we may want to go with links in the short term, at least.

It's important this doc is in place before we go live on Monday, March 1. 

### Are you making a change to site code?

No

### CODE REVIEW

**Comment by jerelmiller** [comment]
@paperclypse I'd love to get an idea of what you'd like to see with the buttons. I would agree, they are a sore spot when used in docs right now, so it would be great to do something about it.

**Comment by paperclypse** [comment]
@jerelmiller I think if the buttons just had some slightly better padding (some on the left and right, and more consistent padding on the top and bottom) that would go a long way.

If they just matched what we've got on the old Docs site, I'd be happy with that: https://docs.newrelic.com/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
+---
+title: New Relic guided install overview
+tags:
+  - Full-Stack Observability
+  - Observe everything
+  - Get started
+contentType: page
+template: basicDoc
```
Comment (bradleycamacho) [inline src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx:8]: These aren't necessary.

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+<ButtonLink
+  role="button"
+  to="https://one.newrelic.com/launcher/nr1-core.home?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5ob21lLXNjcmVlbiJ9&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGlvbkluZGV4IjoxfQ=="
+  variant="primary"
+>
+  Guided install
+</ButtonLink>
+
+If you're hosted in the EU:
+
```
Comment (bradleycamacho) [inline src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx:23]: I'm not a fan of how these render on the site. If you're going to have the "If you're hosted in the eu" line, I'd do a similar line for the other button.

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx (hunk 1 above)
Comment (bradleycamacho) [reply src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx]: Removed these in my commit below.

**Comment by bradleycamacho** [comment]
@paperclypse This looks good! Two notes:

I removed the extra front matter from the doc. Basic docs don't need any front matter designating them as such, since the default is already `basicDoc`. Also, `contentType` and `template` are outdated, the correct front matter field is `type`

The buttons are a bit wonky. I would either remove the EU line, or add a line above the first button. If you do the second option, I would make sure it goes something like:

```
If you're hosted in the Us:
Button
//New line
If you're hosted in the EU:
Button
```

The blank line between the text and button throws me off a bit.

**Comment by paperclypse** [comment]
@bradleycamacho Good call on the buttons. Thanks!

**Comment by paperclypse** [comment]
@bradleycamacho I replaced the buttons with a tip callout and a couple of links.

**Pull request merged**